lib/environment.sh

```diff
@@ -73,6 +73,9 @@ write_ci_profile() {
 write_export() {
   local bp_dir="$1"
   local build_dir="$2"
-  echo "export PATH=\"$build_dir/.heroku/node/bin:$build_dir/.heroku/yarn/bin:\$PATH:$build_dir/node_modules/.bin\"" > $bp_dir/export
-  echo "export NODE_HOME=\"$build_dir/.heroku/node\"" >> $bp_dir/export
+
+  if [ -w ${bp_dir} ]; then
```

Comment:
Actually, can you leave a comment here about why it's necessary before merging?

Reply:
@jmorrell done. i'm also trying to make the directory writable during `local:build`